docs/guides/preinstall.md

```diff
@@ -13,68 +23,109 @@ In macOS Disk Utility, [create a partition](https://support.apple.com/guide/disk
+We need to disable it:
 
-Apple's Secure Boot implementation does not allow booting anything other than macOS and Windows when Secure Boot is enabled (not even shim signed GRUB).
+1. Turn off your Mac
+2. Turn it on and rapidly press and hold `Command-R` until the black screen flashes
+3. Your Mac will boot in the macOS Recovery
+4. Select your user and enter your password
+5. Now, from the menu bar choose Utilities > Startup Security Utility
+6. Enter again the password
+7. Once in Startup Security Utility:
 
-1. Follow [this article's](https://support.apple.com/HT208198) instructions to boot into macOS Recovery and open Startup Security Utility.
-2. Once in Startup Security Utility, set Secure Boot to "No Security" and select "Allow booting from external or removable media". This will allow you to boot from a Linux install ISO.
+   - set Secure Boot to **No Security**
+   - set Allow Boot  Media to **Allow booting from external or removable media**
+
+Now you are able to boot from a Linux install ISO.
 
 !!! Note "Keeping your Mac secure while Secure Boot is off"
-    If you are worried about the reduced security, in Startup Security Utility you can select "Turn On Firmware Password" to require entering a password to boot anything other than the default OS (Currently that would be macOS). After installing Linux on the internal SSD, you have the option to reselect "Disallow booting from external or removable media" in Startup Security Utility.
+    If you are worried about the reduced security, in Startup Security Utility you can select "Turn On Firmware Password" to require entering a password to boot anything other than the
+    default OS. Additionally, after installing Linux on the internal SSD, you will have the option to reselect "Disallow booting from external or removable media" in the Startup Security
+    Utility.
 
 ## Booting your Linux install USB
 
 1. Ensure the Linux Installation USB you created is plugged into your Mac.
 2. Reboot while holding down the option (⌥) key, this will put you in macOS Startup Manager.
-3. Select the orange EFI option with arrow keys and press return/enter on it. If there are two, try the one to the very right first.
-4. You may be put into a Bootloader Menu, it may select a default option if no keys are pressed, but you can use arrow keys to select a different option if you want. They should be labeled, but if you are unsure, check if there is any additional advice on your distro's installation page on this wiki.
+3. Select the orange EFI option with arrow keys and press return/enter on it. If there are two, select the one to the very right (the last one).
+4. The installation may start right away or you may be put into a Bootloader Menu where it may select a default option e.g. "Try or Install Ubuntu", but you can use arrow keys to select a different option if you want.
```

Comment:
What about the instruction stating to refer to the distros installation page?

Reply:
I reverted the missing piece of the distro installation page. [89f231d](https://github.com/gianmaticus/wiki/commit/89f231dafc5bd679bcb3e0ac4be78c1bcd82d1e5)